api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTestsWithScheduledServices.java

```diff
@@ -68,6 +70,25 @@ void shouldRequestNewCardAndSendEmailForSuccessfulClaim() throws JsonProcessingE
         wiremockManager.assertThatDepositFundsRequestMadeForPayment(payment);
     }
 
+    @Test
+    void shouldSendWeWillLetYouKnowEmailForAddressMismatch() throws JsonProcessingException, NotificationClientException {
```

Comment:
Did you also want to check that the letter got sent here too?

Reply:
Added a check for the letter being sent